Gadget/ListExModule/ListExModule.pbi

```diff
@@ -5402,3 +6167,12 @@ CompilerIf #PB_Compiler_IsMainFile
   EndIf
   
 CompilerEndIf
+
+; IDE Options = PureBasic 5.71 LTS (Windows - x64)
+; CursorPosition = 6087
+; FirstLine = 887
+; Folding = EBAAAAAAAAAAAAAAAQAKAAxBAAYDAxBAAAAAAAAGJ9BAAAAAAI9
+; Markers = 587,3164
+; EnableXP
+; DPIAware
+; EnableUnicode
```

Comment:
The PB settings at the end of code files change very frequently, for example, the last cursor position. These changes are almost always unimportant, so these PB settings should not be included in the code files.

In addition, there should be an empty line at the end of each code file. Many Linux programs otherwise do not work correctly.